swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -522,13 +479,13 @@ function initializeHttpClient(string url) returns http:Client|error => new (url)
 
 function initializeEmailClient() returns email:SmtpClient|error => new (host, username, password);
 
-service /healthcare on new http:Listener(port) {
+service /healthcare on new http:Listener(8290) {
 
     resource function post categories/[string category]/reserve(ReservationRequest payload)
             returns http:Created|http:NotFound|http:InternalServerError {
 
         ReservationRequest {
-            patient: {cardNo, ...patient},
+            patient,
```

Comment:
Why do we need to change the source here?

Reply:
I think this is due to the bug right? This causes errors in "diagram view" and "try-out".

Reply:
Yes this change was added to fix the error in `diagram view` and `try-out`